internal/provider/client.go

```diff
@@ -78,6 +78,23 @@ func (c *LumosAPIClient) MakeRequest(method string, endpoint string, requestBody
 
 }
 
+func (c *LumosAPIClient) getUser(id string) (*lumosAPIUserResponse, error) {
```

Comment:
This is a little of a chicken-and-egg problem, because the user will still need to use the Lumos API to get the user's ID.

I would suggest we allow for looking up a user via ID _or_ their email address